### ISSUE TITLE
Help please! I'm going crazy - How to get weak camera perspective working with pytorch3d

### ISSUE THREAD

**Issue by gt732** [dancasas/SMPLitex#9]
Hi,

Thanks for the hard work here. I have everything working with applying the texture to the SMPL model the issue I'm facing is getting the camera perspective correct. I'm using PARE to extract SMPL data from a reference video. PARE uses weak camera perspective, I want to render the exact perspective of the original video used to get the SMPL data.

FROM PARE documentation - https://github.com/mkocabas/PARE
pred_cam (n_frames, 3)          # weak perspective camera parameters in cropped image space (s,tx,ty)
orig_cam (n_frames, 4)          # weak perspective camera parameters in original image space (sx,sy,tx,ty)

How can I convert this into pytorch3d camera? I tried for many hours/days but the camera is always incorrect.

![image](https://github.com/user-attachments/assets/3d955640-5896-47c9-a6c6-940eaeef183e)

Ref video frame 1920x1080

![image](https://github.com/user-attachments/assets/8082d0c2-5441-4f01-b586-43475dc554e4)

Any code references you can point me to? This is the last hurdle I'm facing!

Thanks!!

**Comment by gt732** [comment]
Fixed.

**Comment by dancasas** [comment]
Sorry I couldn't jump earlier to this issue, here's how I set up the PyTorch3D cameras, which should render the SMPL meshes upright facing frontal. 

https://github.com/dancasas/SMPLitex/blob/0476822a8273f96ee8ab3463560d6968c46697c7/scripts/utils/renderer/pytorch3d_renderer.py#L240-L246

How did you fix your issue?

**Comment by gt732** [comment]
@dancasas Np, thanks for the reply! I was able to fix it by using another library mmhuman3d. They have a ton of helper function that set up the correct camera for you when you pass the cam params from VIBE.

https://github.com/open-mmlab/mmhuman3d/blob/8c95747175f8c4bd76d5e0be7be244f8a9a4a6de/mmhuman3d/core/visualization/visualize_smpl.py#L1129

I was able to use the texture provided by this library doing the rendering process in mmhuman3d. 

This is the results!

https://github.com/user-attachments/assets/1cb7dd47-adf3-4146-a199-50263626f2dd



https://github.com/user-attachments/assets/dbd8fb89-487e-459a-ad60-dce6b0d3e5a3
